WickedEngine/wiGraphicsDevice_Vulkan.cpp

```diff
@@ -2283,10 +2267,7 @@ using namespace Vulkan_Internal;
 		allocatorInfo.physicalDevice = physicalDevice;
 		allocatorInfo.device = device;
 		allocatorInfo.instance = instance;
-		if (features_1_2.bufferDeviceAddress)
```

Comment:
I think this should still be checked for

Reply:
Sure? Even if it is core feature in vulkan 1.2?

Reply:
It's not an extension any more but the device will not necessarily support it, that's why you can check support in the features_1_2